messaging/coap/coap.c

```diff
@@ -894,13 +892,11 @@ coap_oscore_parse_outer_option(coap_packet_t *packet,
   }
 #if 0
   case COAP_OPTION_PROXY_SCHEME:
-#if COAP_PROXY_OPTION_PROCESSING
     packet->proxy_scheme = (char *)current_option;
     packet->proxy_scheme_len = option_length;
-#endif
-    OC_DBG("Proxy-Scheme NOT IMPLEMENTED [%.*s]", (int)packet->proxy_scheme_len,
+    OC_DBG("Proxy-Scheme [%.*s]", (int)packet->proxy_scheme_len,
            packet->proxy_scheme);
-    return PROXYING_NOT_SUPPORTED_5_05;
+    return COAP_NO_ERROR;
```

Comment:
why it is changed from `PROXYING_NOT_SUPPORTED_5_05` ?

Reply:
Probably copy&paste artifact, but the whole option ifdef-ed out anyway, if we ever enable proxying then this code will be changed.

Reply:
pls move it back maybe in future it will be needed by someone from OCF

Reply:
Reverted, though mantaining disabled code seems like a bit of a waste of time to me, I'd rather remove it.